client/checks.go

```diff
@@ -172,3 +172,38 @@ func (client *Client) doMultiCheckAction(actionName string, checks []string) (*C
 
 	return results, nil
 }
+
+type RefreshCheckOptions struct {
+	// Name of the check to refresh, required.
+	Name string
+}
+
+type RefreshCheckResult struct {
+	Info *CheckInfo `json:"info"`
+	// The error message, empty if success.
+	Error string `json:"err"`
```

Comment:
Let use the JSON tag "error":
```suggestion
	Error string `json:"error"`
```